apps/api/src/models/ApplicationData.py

```diff
@@ -38,6 +39,7 @@ class RawApplicationData(BaseApplicationData):
 
     first_name: str
     last_name: str
+    application_type: str
```

Comment:
Suggestion: by using a more specific type (e.g. `Literal["Hacker", "Mentor", "Volunteer"]`), Pydantic will automatically handle the validation for us.